source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -393,6 +410,22 @@ export class Carousel extends Container {
             carousel.autoplay?.start();
         });
 
+        carousel.on('navigationNext',  () => {
+            raiseCarouselEvent(this, Enums.CarouselEventType.NextNavigationInteractionType);
```

Comment:
It's a good idea to reuse the event name which slide is using. For example: Enums.CarouselEventType.NavigationNext, Enums.CarouselEventType.NavigationPrev

Reply:
okay